build/webpack/webpack.files.js

```diff
@@ -76,6 +76,12 @@ webpackConfigs[0].plugins.push(new CopyWebpackPlugin([
 		force: true,
 		ignore: ["*.js"]
 	},
+	{
+		from: './src/clients/',
```

Comment:
There is a webpack.services.js you should look at. It isn't currently used because we didn't support customer services.